getting-started/Vulnerability-Scan.md

```diff
@@ -0,0 +1,65 @@
+
+Before beginning the pre-scan process, you will need to set up a Microsoft Azure Portal account to access Defender, if you don’t have an account already. Access to Defender is a paid resource. If you do not have an Azure portal account, you can create one at [Azure portal](https://portal.azure.com).
+
+### Important Notes to Keep in Mind 
+* CVEs are common vulnerabilities and exposures of publicly disclosed computer security flaws. During the pre-scan process you will use the CVE link to identify the vulnerability and remediate it.  
+* Once your image is scanned, in the pop-up blade you will see a CVSS score and if this cvss 3.0 base score is >7 the image will be blocked until remediated. Below is a chart outlining the severity and score range of vulnerabilities.  
+
+<p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/949fb0ac926aa16df4e3ec421a95a5f4960c7658/getting-started/images/cvss3.png" alt="cvss3" width="500"/>
+
+### Vulnerability Pre-Scan Steps 
+1. Sign into your Microsoft Azure Portal Account  
+
+<p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/71c79359fb7b14e5936483559038abb057ceb95c/getting-started/images/step1-azureportal.png" alt="step1-azureportal" width="800"/>
+
+2. In the search bar, enter container registries  
+
+ <p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/71c79359fb7b14e5936483559038abb057ceb95c/getting-started/images/step2-search.png" alt="step2-search" width="800"/>
+
+3. Select your container registry that you have already packaged and uploaded (If you need help uploading the package please refer to here: [Prepare your Azure container technical assets for a Kubernetes application | Microsoft Learn](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=windows) and the additional links at the end of this document). Once selected your view should look something like this: 
+  
+<p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/71c79359fb7b14e5936483559038abb057ceb95c/getting-started/images/step3-registry.png" alt="step3-registry" width="800"/>
+
+4. On the left-hand side of the blade, locate “Microsoft Defender for Cloud” or on the top you can select “visit Microsoft Defender for Cloud”  
+
+<p align="center"><img src="https://github.com/bobmital/kubernetes-offer-samples/blob/71c79359fb7b14e5936483559038abb057ceb95c/getting-started/images/step4-defender.png" alt="step4-defender" width="800"/>
+
+5. You will see the Recommendations pop up under “description” below. Here is where you will be able to view each vulnerability.  
```

Comment:
Say, the customer enabled Defender for registry for the first time, how long does the defender take to show vulnerability results? 
If Defender takes some time to scan and show results, its good to callout those details.

Reply:
@maanasagovi

Reply:
@RamyasreeChakka Will double check on the timeline and will add that here :)